server/src/internalClusterTest/java/org/opensearch/indices/settings/UpdateSettingsIT.java

```diff
@@ -174,8 +175,8 @@ protected Settings nodeSettings(int nodeOrdinal) {
     }
 
     public void testUpdateDependentClusterSettings() {
-        IllegalArgumentException iae = expectThrows(
-            IllegalArgumentException.class,
+        SettingsException iae = expectThrows(
```

Comment:
There are several places in this PR where the variable name "iae" no longer makes sense.